modules/gapi/include/opencv2/gapi/imgproc.hpp

```diff
@@ -120,8 +120,11 @@ namespace imgproc {
             GAPI_Assert(in_uv.chan == 2);
             GAPI_Assert(in_y.depth == CV_8U);
             GAPI_Assert(in_uv.depth == CV_8U);
+            // We should check that uv width is even
+            // for the reason it must contain 2 pixels (u and v)
+            // for each output pixel
```

Comment:
UV plane is a two-channel Mat. Regardless its width in pixels, every pixel _always_ contains two elements (U and V).

Reply:
You are right. uv.width assert is also wrong